### ISSUE TITLE
GraphiQL broken in production. Duplicate "graphql" modules.

### ISSUE THREAD

**Issue by a-barbieri** [BindaCMS/binda-api#11]
As @lufutu suggested is not possible to use GraphiQL in production. See [this comment](https://github.com/BindaCMS/binda-api/issues/9#issuecomment-439341825).

# Bug

Accessing GraphiQL page I see only the `loading...` message and get this error:

```
Uncaught Error: Cannot use t "__Schema" from another module or realm.

Ensure that there is only one instance of "graphql" in the node_modules
directory. If different versions of "graphql" are the dependencies of other
relied on modules, use "resolutions" to ensure only one version is installed.

https://yarnpkg.com/en/docs/selective-version-resolutions

Duplicate "graphql" modules cannot be used at the same time since different
versions may have different capabilities and behavior. The data from one
version used in the function from another could produce confusing and
spurious results.
```

Which then create a second obvious error:

```
Uncaught ReferenceError: GraphiQL is not defined
```

# How to reproduce it

1. create a new Rails `5.2` application
1. install Binda `0.1.10` and Binda API `master`
1. deploy to production (on Heroku or anywhere with Capistrano)
1. go to `http://your.domain.com/admin_panel/graphiql`
1. check your browser console logs

# Hint

The issue is probably due to [GraphiQL Rails](https://rubygems.org/gems/graphiql-rails) or [GraphQL](https://rubygems.org/gems/graphql) as they are the only responsible for injecting `graphql` as a dependency.

# Versions

Rails 5.2
Binda 0.1.10
Binda API master (https://github.com/BindaCMS/binda-api/commit/c0c688494566e6689976cd38faff1aceb1314f70)

**Comment by a-barbieri** [comment]
I can confirm that the issue isn't present when using graphql-rails `1.4.11`.

**Comment by a-barbieri** [comment]
It's possible to temporarily fix this issue using version `v0.1.4.alpha.1`

```ruby
gem "binda-api", github: "BindaCMS/binda-api", tag: "v0.1.4.alpha.1"
```

Just remember to change `config/environments/production.rb` from `config.assets.js_compressor = :uglifier` to `config.assets.js_compressor = Uglifier.new(harmony: true)`
